openlibrary/core/search.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import absolute_import
+
+import urllib
+import urllib2
```

Comment:
https://stackoverflow.com/a/2792652/837575
The `2to3` should help us here, otherwise check ImportError and use urllib instead of urllib2 for python3.

We hit this in several places in the code (mostly uses urllib2)

Reply:
Ah; if it's in multiple places, then I wouldn't worry about it. We can fix it in a sweep. I'll open a new issue for this.

Reply:
or I guess we don't need an issue if it gets fixed automatically :P